src/tutorials/collections/index.md

```diff
@@ -1,161 +1,205 @@
+# Collections combine datasets to minimize clutter
 
-## Creating a paired dataset collection
+A typical multi-dataset analysis of next-generation sequencing (NGS) data usually involves a large number of sequence datasets, such as, for example, fastq datasets generated by an Illumina machine or downloaded from a Short Read Archive. So it's usually a *collection* of similar things that need to be processed in a *collective* fashion:
 
-If you imported [history](https://test.galaxyproject.org/u/anton/h/collections-1) as described [above](/tutorials/collections/#getting-data), your screen will look something like this:
+|      |
+|------|
+|![](/src/tutorials/collections/simpleCollection.png)|
+|<small>**A collection** is any number of datasets bundled as a **single** entity. In this example, to, say, map *N* fastq files against the human genome you need to manually start *N* mapping jobs. Yet you first combine individual datasets into a collection, you will only need start mapping job **once** using the collection as input.</small>|
 
-![manyDatasets](/src/tutorials/collections/manyDatasets.png)
+## Collections reflect data complexity
 
-Now click the checkbox in <img src="/src/tutorials/collections/historyItemControls.png"> and you will see your history changing like this:
+Previous image shows how multiple fastq datasets can be combined in a single collection. But what if the sequencing data one wants to analyze is from a paired-end (or mate-pair) run where each individual sample is represented by *two* fastq files: forward and reverse? Galaxy collections can accommodate this structure:
 
-![historyWithCheckboxes](/src/tutorials/collections/historyWithCheckboxes.png)
+|      |
+|------|
+|![](/src/tutorials/collections/pairedCollection.png)|
+|<small>**Paired collection** preserves the relationship between read pairs and their respective samples.</small>
 
-Let's click **All**, which will select all datasets in the history, then click **For all selected...** and finally select **Build List of Dataset Pairs** from the following menu:
+# Creating collections in practice
 
-![buildPairs](/src/tutorials/collections/buildPairs.png)
+Below we explain how collections can be used in Galaxy. We start with an example where datasets are already in your history. Later in this tutorial we explain how to upload data into collections directly.
 
-The following wizard will appear:
+## Creating a collection from datasets in your history
 
-![collectionCreation](/src/tutorials/collections/collectionCreation.png)
+In Galaxy's language there are two interchangeable terms: collection and list. This is because the simplest collections is just a list of datasets.
```

Comment:
A list is a specific kind of collection so I don't really agree with this statement. You can have a dataset pair and that is a collection but it is not a list.